src/games/strategy/triplea/ai/fastAI/FastAI.java

```diff
@@ -9,14 +9,16 @@
  */
 public class FastAI extends ProAI {
 
+  // Odds estimator
+  private final static IOddsCalculator estimator = new FastOddsEstimator();
+
   public FastAI(final String name, final String type) {
     super(name, type);
   }
 
   @Override
-  public void initializeBattleCalculator() {
-    final IOddsCalculator calc = new FastOddsEstimator();
-    ProBattleUtils.setOddsCalculator(calc);
+  protected void initializeCalc() {
+    calc = new ProOddsCalculator(estimator);
```

Comment:
The code that initalizes these classes already know about that detail. It's the reason for having the different object. Violation of SRP, no unit tests..